functests/ocs/storagecluster_creation_test.go

```diff
@@ -32,7 +31,7 @@ func initSCCreation() (*SCCreation, error) {
 }
 
 func (sccObj *SCCreation) populateDuplicateSC() error {
-	defaultStorageCluster, err := deploymanager.DefaultStorageCluster()
+	defaultStorageCluster, err := tests.DeployManager.DefaultStorageCluster()
```

Comment:
I think this hunk is in the wrong commit - it should be in the "use global deployManager" commit for all I can tell.